tests/unit/test_sql_execution.py

```
@@ -940,3 +940,88 @@ def test_federated_auth_params_bigquery_missing_params(
+        """Test that no action is taken when mssql_trusted_connection is False."""
+        from deepnote_toolkit.sql.sql_execution import _handle_mssql_trusted_connection
+
+        sql_alchemy_dict = {
+            "url": "mssql+pymssql://user:password@myserver/mydb",
+            "params": {"mssql_trusted_connection": False},
+        }
+        original_url = sql_alchemy_dict["url"]
+
+        _handle_mssql_trusted_connection(sql_alchemy_dict)
+
+        self.assertEqual(sql_alchemy_dict["url"], original_url)
```

Ruff (0.14.11)

Use a regular `assert` instead of unittest-style `assertEqual`

Replace `assertEqual(...)` with `assert ...`

(PT009)